wordcounter.rb

```diff
@@ -0,0 +1,30 @@
+class Counter
+  attr_accessor :dictionary, :filepath
+
+  def initialize(filepath)
+    @filepath = filepath
+    @dictionary = {}
```

Comment:
If you instantiate `@dictionary` as a Hash with a default value of zero like so: `Hash.new(0)`, you don't need to check if a given word is already in `@dictionary` before incrementing its count.